ClientLib/src/main/java/com/o3dr/android/client/data/tlog/callback/TLogIteratorCallback.java

```diff
@@ -0,0 +1,24 @@
+package com.o3dr.android.client.data.tlog.callback;
```

Comment:
@chaviw let's have all callbacks and interfaces in the `tlog` package. Let's delete the `callback` package.